core/graph/src/main/java/uk/gov/gchq/gaffer/graph/GraphConfig.java

```diff
@@ -91,36 +100,6 @@ public void setHooks(final List<GraphHook> hooks) {
     }
 
     @Override
-    public boolean equals(final Object obj) {
```

Comment:
View, GraphHooks and GraphLibrary don't implement equals so having a GraphConfig.equals method doesn't really work.